rows.go

```diff
@@ -65,11 +66,19 @@ func (rows *mysqlRows) Columns() []string {
 }
 
 func (rows *mysqlRows) Close() (err error) {
+	if f := rows.finish; f != nil {
```

Comment:
Just wondering, do we actually need to separately track `rows.finish` or would `mc.finish` maybe already be enough?